risc0/zkp/accel/backend/cpu/impl.cpp

```diff
@@ -130,7 +130,7 @@ void batchEvaluateAny(AccelConstSlice<Fp> coeffs,
     size_t id = reinterpret_cast<const uint32_t*>(which.buf()->buf)[i];
     Fp4 x = reinterpret_cast<const Fp4*>(xs.buf()->buf)[i];
     const Fp* coeffLocal = coeffsPtr + (1 << po2) * id;
-    for (size_t i = 0; i < (1 << po2); i++) {
+    for (size_t i = 0; i < ((size_t)1 << po2); i++) {
```

Comment:
I generally try to avoid C-style casts because it's harder to know which kind of cast the compiler is using.